apps/testrunner/test_suites/simple.py

```diff
@@ -21,6 +23,17 @@ def __init__(self, *args, **kwargs):
         self.DEPLOY_HOST = test_runner_config['deploy_host']['hostname']
         self.TARGET_ENV = test_runner_config['target_hosts'][0]['hostname']
 
+    def run_selenium_tests(self):
+        logger.info('Running selenium task...')
+        simple_selenium_test = MainPageSeleniumTest()
+        run_selenium_test_action = RunSeleniumTest()
+        run_selenium_test_action.run(simple_selenium_test)
```

Comment:
No it doesn't :stuck_out_tongue: 

It's a question of on which level to define the scenarios. I think it would be good to enable such generic test creation semantics as you presented, but I would put the definitions of them to a separate class, as otherwise the SimpleTestSuite will grow too much. It touches the subject of how flexible we want to be with our test definition and how do we want to control it - I think we should talk about it offline.